Classes/Domain/Repository/AbstractRepository.php

```diff
@@ -34,4 +34,11 @@ public function crossClientFindAll($returnRawQueryResult = TRUE) {
         return $query->execute($returnRawQueryResult);
     }
 
-}
\ No newline at end of file
+    public function crossClient($active = false)
```

Comment:
Uncritical, but either the name of the function should be more revealing or it at least needs a proper documentation block.